### DISCUSSION TITLE
Plex Discover / Universal Search

### DISCUSSION THREAD

**Discussion by ScoBoXxX** [CodyBerenson/PGMA-Modernized#153] [General]
Seems today Plex announced these new features - Universal Search across all streaming platforms.
How cool would it be to be able to add adult services to the sources?  🍆🍑

**Comment by CodyBerenson** [comment]
@ScoBoXxX 
Agreed!  According to an article, "Over time, Plex wants to open up its universal search, watchlist and discovery **to its API, and enable developers to come up with third-party tools** that Plex itself may not be able to build. "

https://www.protocol.com/entertainment/plex-universal-search-watchlist-discovery

So, it could be that in the future we enable "what's new and trending at GEVI or AEBN or WayBig" to integrate with Plex's Discover.  :)

To me, even more importantly on the non porn side, Discover's Watchlist integrates seamlessly with Sonarr (for sure, and I'm next going to try with Radarr), so as one is made aware of Streaming TV Series that they may be interested in, by adding the series to the Plex Watchlist, Sonarr will do its magic to add to the Plex library when it becomes available...for those folks who may already use Sonarr or considering for their Plex TV Series library upkeep.
